src/containers.js

```diff
@@ -0,0 +1,142 @@
+import {bytesToString, toUint8} from './byte-helpers.js';
+
+export const getId3Offset = function(bytes) {
+  bytes = toUint8(bytes);
+
+  if (bytesToString(bytes.subarray(0, 3)) !== 'ID3') {
+    return 0;
```

Comment:
This is assuming that `bytes` always contains valid ID3 data?

Reply:
if it doesn't then we likely won't be able to determine the type.